pkg/render/common.go

```diff
@@ -293,3 +293,12 @@ func secretsToRuntimeObjects(secrets ...*v1.Secret) []runtime.Object {
 	}
 	return objs
 }
+
+func GetStandardTolerations() []v1.Toleration {
```

Comment:
I think maybe something like GetNoExNoSchedTolerations or GetCriticalTolerations would be more descriptive. WDYT?